app/controllers/APIController.scala

```diff
@@ -0,0 +1,39 @@
+package controllers
+
+import java.time.LocalDateTime
+import javax.inject._
+import play.api._
+import play.api.mvc._
+import play.api.libs.json._
+import play.api.Environment._
+
+import java.time.format.DateTimeFormatter
+
+/**
+ * This controller creates an `Action` to handle HTTP requests to the
+ * application's home page.
+ */
+@Singleton
```

Comment:
This comment is non descriptive, please remove it

Reply:
comment was removed